helix-rest/src/test/java/org/apache/helix/rest/clusterMaintenanceService/TestMaintenanceManagementService.java

```diff
@@ -114,7 +114,7 @@ public MockMaintenanceManagementService(ZKHelixDataAccessor dataAccessor,
 
     @Override
     protected Map<String, Boolean> getInstanceHealthStatus(String clusterId, String instanceName,
-        List<HealthCheck> healthChecks) {
+        List<HealthCheck> healthChecks, Set<String> toBeStoppedInstances) {
```

Comment:
The reason why we still need to change this override is the `getInstanceStoppableCheck` will eventually call `protected Map<String, Boolean> getInstanceHealthStatus(String clusterId, String instanceName,
        List<HealthCheck> healthChecks, Set<String> toBeStoppedInstances) `